oras/auth/token.py

```diff
@@ -44,7 +44,11 @@ def reset_basic_auth(self):
             self.set_header("Authorization", "Basic %s" % self._basic_auth)
 
     def authenticate_request(
-        self, original: requests.Response, headers: dict, refresh=False
+        self,
+        original: requests.Response,
+        headers: dict,
+        refresh=False,
+        skipAnonToken=False,
```

Comment:
Please use `skip_anon_token` to match the rest of the code base style, and since CamelCase is typically for Go/Java.

Reply:
oops sorry 😅  will do 👍

Reply:
done with 102381c